### ISSUE TITLE
gdbtrace.init does not seem to work with newer version of arm-none-eabi-gdb

### ISSUE THREAD

**Issue by ClemensElflein** [orbcode/orbuculum#120]
I have tried using orbuculum with the toolchain linked here: https://modm.io/guide/installation/

The issue is that the arm-none-eabi-gdb (version 12.1.90) does not correctly process the `gdbtrace.init` from this repository and therefore the enableSTM32TRACE command cannot be used.

Example output is here: https://cdn.discordapp.com/attachments/614885210395508738/1146444609174654987/image.png

**Comment by salkinium** [comment]
Weird, which platform did you try this on? Windows perhaps?
I use both the ARM (non-Python) and the xpack (Python3) GDB v12 on macOS and it works fine.

**Comment by ClemensElflein** [comment]
PopOS 22.04, so basically Ubuntu 22.04

**Comment by niklaut** [comment]
This exact problem is now also happening with my GDB after I updated the `gdbtrace.init` to the latest version.

**Comment by zyp** [comment]
Which platform are you on? I haven't been able to reproduce it yet.

**Comment by niklaut** [comment]
I'm on macOS 13.5.2 with xpack arm-none-eabi-gdb-py3
```
GNU gdb (xPack GNU Arm Embedded GCC arm64) 12.1.90.20221210-git
```

It's weird, cos parts of the script are shown with autocompletion:

```
(gdb) _s
_setAddressesEFR32MG12  _setAddressesETM4       _setAddressesIMXRT      _setAddressesNRF        _setAddressesNRF52      _setAddressesNRF53      _setAddressesSTM32      _setAddressesTM4C       _startETMv35            _startETMv4
(gdb) start
start     startETM  starti
(gdb) stop
stop     stopETM
(gdb) describeETM
(gdb) prepa
```
but then it stops after `describeETM`, so something in it breaks it. (there's a duplicate stopETM function also btw)
if I remove the `describeETM` function, I can call the `enableSTM32TRACE` function.

I can see that it configures the pins, but the orbtrace's trace led doesn't turn red. I suspect that this is a hardware issue with my wiring though.

**Comment by zyp** [comment]
Thanks, reproduced and fixed in 7809380. `describeETM` was missing an `end` at the end, so it ate the rest of the script, and the way it was indented didn't make it obvious.

**Comment by niklaut** [comment]
Yup, that fixed it for me, thanks!
